python/tk_multi_workfiles/actions/open_file_action.py

```diff
@@ -180,26 +201,31 @@ def _open_in_current_work_area(self, src_path, src_template, file, src_work_area
         src_version = None
         dst_version = None
         if "version" in dst_work_area.work_template.keys:
-            # need to figure out the next version:
-            src_version = fields["version"]
 
-            # build a file key from the fields: 
-            file_key = FileItem.build_file_key(fields, 
-                                               dst_work_area.work_template, 
-                                               dst_work_area.version_compare_ignore_fields)
-    
-            # look for all files that match this key:
-            finder = FileFinder()
-            found_files = finder.find_files(dst_work_area.work_template, 
-                                            dst_work_area.publish_template, 
-                                            dst_work_area.context, 
-                                            file_key)
+            if self._next_version_override is not None:
```

Comment:
The diff is terrible here for some reason. I've simply moved the block above in the `else` clause and allowed to use the override if set.